### ISSUE TITLE
Handle background layers

### ISSUE THREAD

**Issue by pcav** [sbenten/d3MapRenderer#43]
If the project contains background layers (e.g. from QuickMapServices plugin), they are not handled. I suggest replacing them with a direct call to the services.

**Comment by sbenten** [comment]
Yeah, the d3 plugin only works with vector layers.  Raster and slippy map layers aren’t supported as such. 

 

D3 is still more of a visualisation tool than a full web mapping solution.

**Comment by pcav** [comment]
Thanks for the clarification. Please state this clearly in the About.
It would be good also to check upon output creation whether the project contains rasters, services and other layers unmanaged, and issue a notice to the user.
